README.md

```diff
@@ -74,38 +74,55 @@ docker push codeforafrica/ckan:latest
 
 ---
 
-## Deployment (Untested)
+## Deployment
 
 We use [dokku](http://dokku.viewdocs.io/dokku/) for deployment so you'd need to install and set it up first.
 
 Once installed, we can do the following:
-```sh
-# Create the Dokku app
+
+1. Create the Dokku app and add a domain to it
+
+``` 
 dokku apps:create ckan
-dokku domains:add ckan http://openafrica.net
+dokku domains:add ckan openafrica.net
+```
 
-# Letsencrypt
-sudo dokku plugin:install https://github.com/dokku/dokku-letsencrypt.git
+2. Add letsencrypt for free `https` certificate
+
+Install the [dokku-letsencrypt](https://github.com/dokku/dokku-letsencrypt) plugin and set the config variables
+
+```
 sudo dokku letsencrypt ckan
+```
 
-# Solr + Redis
+3. Run Solr + Redis + Postgres
+
+Install the [solr](https://github.com/dokku/dokku-solr), [redis](https://github.com/dokku/dokku-redis), and [postgres](https://github.com/dokku/dokku-postgres) plugins and set the necessary environment variables
+
+```
 sudo dokku plugin:install https://github.com/dokku/dokku-solr.git solr
 sudo dokku plugin:install https://github.com/dokku/dokku-redis.git redis
+sudo dokku plugin:install https://github.com/dokku/dokku-postgres.git postgres
+
 dokku solr:create solr
 dokku redis:create redis
+dokku postgres:create postgres
 dokku solr:link solr ckan
 dokku redis:link redis ckan
+dokku postgres:link postgres ckan
 ```
 
 Once done, you can push this repository to dokku:
 
-```sh
+```
 git remote add dokku dokku@openafrica.net:ckan
 git push dokku
 ```
 
+***NOTE:** Make sure to have the [appropraite permissions to push to dokku](?).*
```

Comment:
Please add link for dokku permission where "?" is.

Reply:
Done